package.json

```
@@ -25,5 +25,17 @@
     "start": "wp-scripts start",
     "deploy:sake": "npm run build:assets && npx sake deploy --deployAssets=0",
     "build": "./bin/build-zip.sh"
-  }
+  },
+  "woorelease": {
+    "version_replace_paths": [
+      "includes",
+      "class-wc-facebookcommerce.php",
+      "facebook-commerce-events-tracker.php",
+      "facebook-commerce-messenger-chat.php",
+      "facebook-commerce-pixel-event.php",
+      "facebook-commerce.php",
+      "facebook-config-warmer.php",
+      "facebook-for-woocommerce.php"
```

I tested using [`./`](https://github.com/woocommerce/facebook-for-woocommerce/commit/83155d7ef05a16b60b99601bc8fd097d13863aba) and I got the same result. I don't mind being explicit either

```suggestion
      "./"
```

---

I have noticed this but I just wanted to exclude the ./vendors folder from processing.

---

that makes sense! 👍🏽